pkg/controller/nodes/task/handler.go

```diff
@@ -298,6 +319,15 @@ func (t Handler) invokePlugin(ctx context.Context, p pluginCore.Plugin, tCtx *ta
 		}
 	}
 
+	if !pluginTrns.IsPreviouslyObserved() {
```

Comment:
should we also make this `t.metrics.pluginPanics.Inc(ctx)` a labeled metric?

Reply:
It is but not tagged at task type level.